cluster/gce/trusty/configure.sh

```diff
@@ -144,8 +144,17 @@ install_additional_packages() {
   fi
 }
 
-# Downloads kubernetes binaries and salt tarball, unpacks them, and places them
-# to suitable directories.
+# Retry a download until we get it.
+#
+# $1 is the file to creat
+# $2 is the URL to download
+download_or_bust() {
+  rm -f $1 > /dev/null
+  curl --ipv4 -Lo "$1" --connect-timeout 20 --retry 6 --retry-delay 10 "$2"
```

Comment:
In files such as cluster/gce/util.sh, we typically use $1, $2. The curly braces are used for named variables